### ISSUE TITLE
2 color contrast issues with reCaptcha

### ISSUE THREAD

**Issue by langyanbin0314** [google/recaptcha#443]
Title: WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (span)
Tags: Accessibility, WCAG 1.4.3, color-contrast

Issue: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (color-contrast - https://accessibilityinsights.io/info-examples/web/color-contrast)

1. Element path: iframe[width="\32 56"];.rc-anchor-invisible-text > span

    Snippet: <span>protected by <strong>reCAPTCHA</strong></span>

2. Element path: iframe[width="\32 56"];strong

    Snippet: <strong>reCAPTCHA</strong>

How to fix: 
Fix any of the following:
  Element has insufficient color contrast of 3.29 (foreground color: #ffffff, background color: #4a90e2, font size: 9.8pt (13px), font weight: normal). Expected contrast ratio of 4.5:1

Environment: Chrome version 88.0.4324.150

====

This accessibility issue was found using Accessibility Insights for Web 2.23.0 (axe-core 4.1.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

**Comment by rowan-m** [comment]
Repo is just for PHP client issues. This is about the reCAPTCHA service.
